src/main/java/lotto/domain/Tickets.java

```diff
@@ -0,0 +1,55 @@
+package lotto.domain;
+
+import java.util.EnumMap;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.atomic.AtomicLong;
+
+public class Tickets {
+
+        private final List<Ticket> values;
+        private EnumMap<Rank, Integer> matchRankCountMap = new EnumMap<>(Rank.class);
```

Comment:
변수의 타입은 상위 타입으로 선언해 주시면 좋을 것 같아요 
```suggestion
        private Map<Rank, Integer> matchRankCountMap = new EnumMap<>(Rank.class);
```

앞서 드린 코멘트와 마찬가지로 누적되는 이슈가 있으니 참고해 주세요

Reply:
말씀 주신 덕분에.. Map 선언에 대해서 고민해보게 됐습니다.
기존에 짜여진 코드보면 대부분 Map 변수 타입은 Map으로.. 그러나 new 생성 시엔 Map의 상속구현체로 구현되어 있는걸 많이 봤었습니다,,
그런데 전 딱히 그 이유를 고민하지않고 '상속구현체 타입 그대로 선언하면 되는 거 아닌가?' 라고 막연하게 생각했었는데요.
오늘 관련 여러자료들을 찾아보니 다형성의 장점을 활용하여 코드의 유지보수성을 높이는 전략이었다는걸 알게 됐습니당 .. ^^

Map 변수 선언 시, 상위타입인 Map으로 선언할 경우,,
Map의 하위클래스는 날이 갈 수록 다양해지고 있는데,, 코드를 유지보수하다가 더 좋은 하위클래스가 나오면 갈아끼우기 용이할 것 같습니당 ㅎㅎ
감사합니다!